starlette/concurrency.py

```diff
@@ -13,9 +13,10 @@
 
 T = typing.TypeVar("T")
 P = ParamSpec("P")
+CoroutineAny = typing.Coroutine[typing.Any, typing.Any, typing.Any]
```

Comment:
Can we make this private `_CoroutineAny`?

Reply:
https://github.com/encode/starlette/pull/2180/commits/b1766d404dbeb6bfb6ba7e99c4bed5ade0683688